### PR TITLE
Updating Getting Started Scenario to OpenShift 3.11

### LINKED ISSUE #439: parksmap-katacoda image does not display map after route created
Site responds, however only with Map Visualizer on OpenShift 3 in the upper left hand corner.  Grey scale and streen buttons appear and the +/- buttons also appear, but no map is displayed.

### LINKED ISSUE #456: Introduction - Getting Started With OpenShift for Developers
I think there's an issue in the first tutorial. I can't see the maps displayed in the app, I don't know why. I did this tutorial few days ago and the problem wasn't there, so maybe it's a recent bug if I can guess.

forgetu

### PR DESCRIPTION
**Pull request by danielhelfand**
Closes #439 
Closes #456

This pull request updates the `Getting Started with OpenShift for Developers` scenario to the OpenShift 3.11 image. This also required screen shots and content to be updated for the 3.11 web console differences. Additionally, the following commands were added to `set-env.sh` to aid with certificates needed for the scenario:

```
curl -LO https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem > /dev/null 2>&1
oc login -u developer -p developer https://master:8443 --certificate-authority=lets-encrypt-x3-cross-signed.pem --insecure-skip-tls-verify=true > /dev/null 2>&1
``` 

The scenario is currently available [here](https://www.katacoda.com/dhelfand/courses/introduction/getting-started) for verification of updates.

This will also fix the parksmap frontend, which is currently using an expired Mapbox key. The image has been updated to use tag `1.2.0`, which uses Openstreet instead of Mapbox.

### CODE REVIEW

**Comment by BenHall** [comment]
Thanks! Let me review and get back to you. 

Do you have a link to your profile where I can test it?

**Comment by danielhelfand** [comment]
@BenHall https://www.katacoda.com/dhelfand/courses/introduction/getting-started

**Comment by danielhelfand** [comment]
@BenHall Appreciate it!

**Comment by GrahamDumpleton** [comment]
Link on initial page of:

```
official https://tools.jboss.org/features/openshift.html plugin.
```

doesn't work.

**Comment by GrahamDumpleton** [comment]
Any references to 'OpenShift Origin' need to be changed to refer to OKD. These occur on multiple pages.

**Comment by GrahamDumpleton** [comment]
In step 2, it says:

```
The tools that you'll need for building and deploying your application can also be found here on the left, as well as setting access controls.
```

Not sure it is valid to say "as well as setting access controls" as the ability to set who are members of project and what role they have are not accessible from those menu entries on left. Not sure of original intent of that statement, but doesn't read right to me.

**Comment by GrahamDumpleton** [comment]
In step 5 on routing of requests, the initial paragraph reads:

```
Services provide internal abstraction and load balancing within an OpenShift environment, as sometimes clients (users, systems, devices, etc.) outside of OpenShift need to access an application. The way that external clients are able to access applications running in OpenShift is through the OpenShift routing layer. The data object behind that layer is a Route.
```

That doesn't entirely make sense in first sentence.

Maybe reword to:

```
Services provide internal abstraction and load balancing within an OpenShift environment, but sometimes clients (users, systems, devices, etc.) outside of OpenShift need to access an application. The way that external clients are able to access applications running in OpenShift is through the OpenShift routing layer. The data object behind that layer is a Route.
```

So change 'as' to 'but'.

**Comment by GrahamDumpleton** [comment]
In step 6 on building from source code, when you click on continue to overview, the page for the new build/deployment is usually going to be collapsed. May need to add a note to say to click on the arrow to left of nationalparks-katacoda name to expand pane if collapsed.

**Comment by GrahamDumpleton** [comment]
On summary page, the link http://www.openshift.org/vm should be replaced with https://www.okd.io/minishift/ as the vm link is an old one for old VirtualBox system that for now at least redirects, but could stop in the future.

**Comment by GrahamDumpleton** [comment]
On summary page, the link:

* https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md

no longer works. I'm actually not sure where they moved it to.

Also talk to @jorgemoralespou about referencing ``oc-cluster-wrapper``. Do we still want to be referencing that?

**Comment by GrahamDumpleton** [comment]
On summary page change link:

* https://www.openshift.com/pricing/index.html

to:

* https://www.openshift.com/products/online/

and change:

* https://www.openshift.com/dedicated/

to:

* https://www.openshift.com/products/dedicated/

**Comment by danielhelfand** [comment]
> On summary page change link:
> 
> * https://www.openshift.com/pricing/index.html
> 
> to:
> 
> * https://www.openshift.com/products/online/
> 
> and change:
> 
> * https://www.openshift.com/dedicated/
> 
> to:
> 
> * https://www.openshift.com/products/dedicated/

This has been addressed via [5721599](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/57215991a6a78983f9315e1038ec6817b5ed4f7b)

**Comment by danielhelfand** [comment]
> On summary page, the link:
> 
> * https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md
> 
> no longer works. I'm actually not sure where they moved it to.
> 
> Also talk to @jorgemoralespou about referencing `oc-cluster-wrapper`. Do we still want to be referencing that?

Jorge said to delete this, so I also removed this via [5721599](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/57215991a6a78983f9315e1038ec6817b5ed4f7b)

**Comment by danielhelfand** [comment]
> On summary page, the link http://www.openshift.org/vm should be replaced with https://www.okd.io/minishift/ as the vm link is an old one for old VirtualBox system that for now at least redirects, but could stop in the future.

Addressed via [5721599](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/57215991a6a78983f9315e1038ec6817b5ed4f7b)

**Comment by danielhelfand** [comment]
> Any references to 'OpenShift Origin' need to be changed to refer to OKD. These occur on multiple pages.

Addressed via [7a57764](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/7a577642809ea5928bb0cbf0adde2b6181aa05ed)

**Comment by danielhelfand** [comment]
> Link on initial page of:
> 
> ```
> official https://tools.jboss.org/features/openshift.html plugin.
> ```
> 
> doesn't work.

Looks like markdown was out of order. Fixed via [b57a032](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/b57a032653265041582c554121b164b11aec2637)

**Comment by danielhelfand** [comment]
> In step 2, it says:
> 
> ```
> The tools that you'll need for building and deploying your application can also be found here on the left, as well as setting access controls.
> ```
> 
> Not sure it is valid to say "as well as setting access controls" as the ability to set who are members of project and what role they have are not accessible from those menu entries on left. Not sure of original intent of that statement, but doesn't read right to me.

Removed via [391a361](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/391a36185936551b934eac35b2930720f5a84a6e)

**Comment by danielhelfand** [comment]
> In step 5 on routing of requests, the initial paragraph reads:
> 
> ```
> Services provide internal abstraction and load balancing within an OpenShift environment, as sometimes clients (users, systems, devices, etc.) outside of OpenShift need to access an application. The way that external clients are able to access applications running in OpenShift is through the OpenShift routing layer. The data object behind that layer is a Route.
> ```
> 
> That doesn't entirely make sense in first sentence.
> 
> Maybe reword to:
> 
> ```
> Services provide internal abstraction and load balancing within an OpenShift environment, but sometimes clients (users, systems, devices, etc.) outside of OpenShift need to access an application. The way that external clients are able to access applications running in OpenShift is through the OpenShift routing layer. The data object behind that layer is a Route.
> ```
> 
> So change 'as' to 'but'.

Addressed via [9f9ab35](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/9f9ab35d9298c2649b4708315fc3387c72b3abf5)

**Comment by danielhelfand** [comment]
> In step 6 on building from source code, when you click on continue to overview, the page for the new build/deployment is usually going to be collapsed. May need to add a note to say to click on the arrow to left of nationalparks-katacoda name to expand pane if collapsed.

Addressed via [2a244e6](https://github.com/openshift-labs/learn-katacoda/pull/499/commits/2a244e657ca6a914e14fffe8c5d16490501bf9b8)

File: introduction/getting-started/intro.md [hunk 1]
```diff
@@ -5,30 +5,29 @@ In this self paced tutorial you will learn how to use the OpenShift Container Pl
...
+Both the command line tool and the web console actually communicate to OpenShift via the same method, the REST API.  Having a robust API allows users to create their own scripts and automation depending on their specific requirements. For detailed information about the REST API, check out the official documentation at: https://docs.openshift.org/latest/rest_api/index.html
 
-During this training, you will be using both the command line tool and the web console.  However, it should be noted that there are plugins for several integrated development environments as well.  For example, to use OpenShift from the Eclipse IDE, you would want to use the official [https://tools.jboss.org/features/openshift.html](JBoss Tools) plugin.
+During this training, you will be using both the command line tool and the web console.  However, it should be noted that there are plugins for several integrated development environments as well. For example, to use OpenShift from the Eclipse IDE, you would want to use the official [JBoss Tools](https://tools.jboss.org/features/openshift.html) plugin.
```
Comment (danielhelfand) [inline introduction/getting-started/intro.md]: Just leaving a note here that we'll want to include more IDE options in the future, but this can be done in a different pr.

**Comment by jorgemoralespou** [comment]
LGTM, LGM
Looks good to me, let's get merged

**Comment by BenHall** [comment]
@jorgemoralespou I need to allocate more infrastructure first. Will have it ready by early next week.
